definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -67,32 +67,6 @@
     "FieldTypeParameter": "ServiceApplicationPaymentTypeEnum",
     "SecurityClassification": "Public"
   },
-  {
-    "LiveFrom": "03/08/2020",
-    "CaseTypeID": "DIVORCE",
-    "ID": "PBANumber",
-    "Label": "Enter your account number",
-    "HintText": "Example: PBA0896124",
-    "FieldType": "TextArea",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "03/08/2020",
```

Comment:
```
{
    “LiveFrom”: “01/01/2017",
    “CaseTypeID”: “DIVORCE”,
    “ID”: “FeeAccountReference”,
    “Label”: “Enter your reference”,
    “HintText”: “This will appear on your statement to help you identify this payment”,
    “FieldType”: “Text”,
    “SecurityClassification”: “Public”
  },
```